### ISSUE TITLE
Flatpak doesnt quack or run if the app is closed

### ISSUE THREAD

**Issue by joshbowyer** [matfantinel/reminduck#58]
Installed the Flatpak on Debian 10 (pinephone/phosh and laptop/cinnamon) and there is no audio for the notification.

**Comment by joshbowyer** [comment]
just built from source and it still doesnt quack (built the deb file using debuild)

**Comment by matfantinel** [comment]
Hah, I think there was a misunderstanding here 😅; It doesn't actually play a quacking sound on notifications (the quack is just in text at the moment)

I've considered adding it during development but was advised not to; however, I still think it would be fun, and it might be something to add soon since I have to port Reminduck over to elementary 6 now.

**Comment by joshbowyer** [comment]
Whoever advised against it has no idea what theyre talking about; a reminder duck needs to QUACK!

**Comment by insanerwayner** [comment]
Yes, I was expecting an audible quack. I think it should at least be a configurable option. For me, I am on multiple devices a lot and will not notice the notification without it being audible. I was using it to remind me to take medications.
